include/common_structs.h

```diff
@@ -1586,6 +1630,83 @@ typedef struct {
     /* 0x1D */ char unk_1D[3];
 } UIPanel; // size = 0x20
 
+// BEGIN ENTITY-SPECIFIC STRUCTS
```

Comment:
Do these need to be global?

Reply:
They're used in each entity's file, along with elsewhere where the entity is used, so yeah.